ttnn/cpp/ttnn/operations/ccl/all_gather/device/multi_core/all_gather_op_multi_core.cpp

```diff
@@ -478,6 +482,9 @@ operation::ProgramWithCallbacks all_gather_multi_core_with_workers_helper(
             /* All gather fusion */
             if (fuse_op) {
                 fused_op_signaler->init_all_gather(program, device, receiver_workers, receiver_worker_cores);
+                if (direction == 1) {
```

Comment:
I don't know what a direction of 0, 1, or 2 is, or whether it could go higher.

What is the range of possible values and what does each direction value mean?

Also, in the future (not for this PR), you may want to have enums/constants to make things more clear.

Reply:
Currently, based on the implementation of ring all gather, there are only 2 directions, CW and CCW, which are 0 and 1 respectively.

enums would definitely help here